src/capwap_loc_provider_sup.erl

```diff
@@ -0,0 +1,57 @@
+%% Copyright (C) 2013-2023, Travelping GmbH <info@travelping.com>
+
+%% This program is free software: you can redistribute it and/or modify
+%% it under the terms of the GNU Affero General Public License as published by
+%% the Free Software Foundation, either version 3 of the License, or
+%% (at your option) any later version.
+
+%% This program is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty of
+%% MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+%% GNU Affero General Public License for more details.
+
+%% You should have received a copy of the GNU Affero General Public License
+%% along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+-module(capwap_loc_provider_sup).
+
+-behaviour(supervisor).
+
+%% API
+-export([start_link/1]).
+
+%% Supervisor callbacks
+-export([init/1]).
+
+%% Helper macro for declaring children of supervisor
+-define(CHILD(I, Type, Args), {I, {I, start_link, Args}, permanent, 5000, Type, [I]}).
+
+-include_lib("kernel/include/logger.hrl").
+
+%% ===================================================================
+%% API functions
+%% ===================================================================
+
+start_link(Config) ->
+    supervisor:start_link({local, ?MODULE}, ?MODULE, [Config]).
+
+%% ===================================================================
+%% Supervisor callbacks
+%% ===================================================================
+
+%%===================================================================
```

Comment:
Please use one code style of comments inside of the one module

Reply:
ok